app/models/project_media.rb

```diff
@@ -69,4 +74,50 @@ def add_elasticsearch_data
     #ElasticSearchWorker.perform_in(1.second, YAML::dump(ms), YAML::dump({}), 'add_parent')
   end
 
+  def cached_annotations(type = nil)
+    self.annotations.where(annotation_type: type)
+  end
+
+  def get_media_annotations(type = nil)
+    em = self.media.annotations.where(annotation_type: type).last
+  end
+
+  def jsondata
```

Comment:
If this is only used by GraphQL `ProjectMedia` type, we can easily have the code there call `to_json` instead of a specialized method just for that.